public_suffix_list.dat

```diff
@@ -13865,6 +13865,10 @@ pcloud.host
 // Submitted by Matthew Brown <mattbrown@nyc.mn>
 nyc.mn
 
+// O3O.Foundation : https://o3o.foundation/
+// Submitted by the prvcy.page Registry Team <psl@registry.prvcy.page>
```

Comment:
Hello @o3o-ca just a heads-up that the email address you left in the comment section is not valid, as the domain of your email address doesn't have MX records, so it won't be reachable if the PSL maintainers or volunteers try to contact you or your organization in the future. It's recommended to either fix the email account or submit an amendment to update it to a valid one. Thank you.